pages/examples/bar.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import * as Plot from '@observablehq/plot';
+import Head from 'next/head';
+
+import { readData } from '../helpers';
+
+interface Letter {
+  letter: string,
+  frequency: number,
+}
+
+interface Props {
+  data: Letter[]
+}
+
+const Histogram: React.FC<Props> = ({ data }) => {
```

Comment:
Let's change this component's name as well:

```suggestion
const BarChart: React.FC<Props> = ({ data }) => {
```